site/README.md

```diff
@@ -19,10 +19,21 @@ limitations under the License.
 
 # Apache Calcite docs site
 
-This directory contains the code for the Apache Calcite web site,
-[calcite.apache.org](https://calcite.apache.org/).
-
-You can build the site manually using your environment or use the docker compose file.
+This directory contains the code and templates for generating the Apache Calcite website,
```

Comment:
I think "codes" might be better.

Reply:
I have always found "code" to be more appropriate. I'll note that I am a native speaker, but of course unaware of whether anyone else is and won't presume that all native speakers agree with me.

Reply:
I used sources/templates instead of code and templates in an attempt to please everybody.